examples/slack/styles/tailwind.css

```
@@ -0,0 +1,27 @@
+/*
+ * Tailwind input for the image-rendered cards. Compile to styles/brand.css with
+ * `pnpm build:css`; that compiled sheet is fed to `createChannel({ render:
+ * { stylesheets } })` and Takumi resolves the classes when rasterizing.
+ *
+ * `@source` tells Tailwind which files to scan for class names (the card
+ * components), so only used utilities land in the output.
+ */
+@import "tailwindcss";
+
+@source "../app/**/*.{ts,tsx}";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)